src/main/java/com/KacFlor/ShopSpring/service/UserService.java

```diff
@@ -17,7 +21,68 @@ public UserService(UserRepository userRepository){
         this.userRepository = userRepository;
     }
 
-    public List<User> getUser(){
+    public List<User> getAllUser(){
         return userRepository.findAll();
     }
+
+    public User getCurrentUser() {
+        Object principal = SecurityContextHolder.getContext().getAuthentication().getPrincipal();
+        String username = ((UserDetails) principal).getUsername();
+
+        return userRepository.findByLogin(username);
+    }
+
+
+    public Boolean changeName(String newName){
```

Comment:
Why Boolean and not bool?

Reply:
bool doesn't exist in java as i know

Reply:
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html are you sure?